src/Mvc/src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/DefaultModelBindingContext.cs

```diff
@@ -184,6 +188,26 @@ public override ModelBindingResult Result
             }
         }
 
+        private int MaxModelBindingRecursionDepth
+        {
+            get
+            {
+                if (_maxModelBindingRecursionDepth == 0)
+                {
+                    var mvcOptions = ActionContext?.HttpContext?.RequestServices?.GetService<IOptions<MvcOptions>>();
+                    if (mvcOptions == null)
+                    {
+                        // Ignore incomplete initialization. This must be a test scenario.
+                        return 32;
```

Comment:
Should this be a constant on `MvcOptions`?

Reply:
:ok: `internal` or `public`?

Reply:
internal